devtools/make_package.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+#
+# Script to make a Caliopen package
+#
+
+BASEDIR="`pwd`/.."
+DISTDIR="${BASEDIR}/dist"
+
+
+function make_python_package() {
```

Comment:
I mean, I think this function should be inside of the package source directory, as it can be if we weren't in a mono-repo.

This script (make_package.sh or it could be make_release) should know witch package to build (thanks to package.yaml) but not how to package each part of the app.  
And each source package directory knows how to package itself.